src/main/kotlin/racingcar/domain/model/Car.kt

```diff
@@ -5,22 +5,26 @@ import racingcar.domain.interfaces.Raceable
 /**
  * 자동차 객체
  * */
-class Car(override val name: String, override var position: Int = DEFAULT_POSITION): Raceable {
+class Car(override val name: String, override var position: Int = DEFAULT_POSITION) : Raceable {
 
     /**
      * 위치 변경
      * */
     override fun move(condition: Int) {
-        require(condition in (MIN_MOVE_VALUE..MAX_MOVE_VALUE))
-        if (condition >= ADVANCE_MIN_VALUE) {
+        if (isMove(condition)) {
             position++
```

Comment:
자동차가 움직이는 이 연산자도 함수를 통해 가독성을 높힐 수 있어요
```suggestion
            move()
```